keyboards/atreus/keymaps/replicaJunction/config.h

```diff
@@ -1,94 +1,17 @@
-/*
-Config file - Atreus QMK with replicaJunction layout
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace the first few lines, eg, these lines: 
```c
#ifndef CONFIG_USER_H
#define CONFIG_USER_H
```
with just `#pragma one`, and remove the endif at the end?